From 52caeddddebf9051fff33bed9e8db80ec9cdd362 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Fri, 3 Apr 2015 19:26:56 +0800
Subject: [PATCH] MLK-10588: ARM: dts: imx7d: enable wdog1 by default

Enable wdog1 as default watchdog device since on all boards we use wdog1.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit d254198e5101b3e9e74dd6b147d77b9546b2f52c)
---
 arch/arm/boot/dts/imx7d.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index bb564af..b394e62 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -464,7 +464,6 @@
 				reg = <0x30280000 0x10000>;
 				interrupts = <GIC_SPI 78 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX7D_WDOG1_ROOT_CLK>;
-				status = "disabled";
 			};
 
 			wdog2: wdog@30290000 {
-- 
1.8.0

